gslb/cache/controller_obj_cache.go

```diff
@@ -126,7 +126,11 @@ func (h *AviHmCache) AviHmCacheGetHmsForGS(tenant, gsName string) []interface{}
 			continue
 		}
 		hmObj := v.(*AviHmObj)
-		if strings.Contains(hmObj.Description, gsName) {
+		descPrefix := nodes.CreatedByAMKO + ", gsname: " + gsName
```

Comment:
@arihantg: Will this work for non-path HM for passthrough Vs where description is only CreatedByAMKO? https://github.com/vmware/global-load-balancing-services-for-kubernetes/blob/master/gslb/nodes/avi_model_nodes.go#L162 or for those HMs we use different criteria?

Reply:
we don't remove non-path HM for passthrough GS since they are shared across all passthrough GS